src/pages/NewGroupPage.js

```diff
@@ -249,14 +250,21 @@ class NewGroupPage extends Component {
                         headerTitle={headerTitle}
                         headerMessage={headerMessage}
                     />
-                    <TouchableOpacity
-                        onPress={this.createGroup}
-                        style={[styles.button, styles.buttonSuccess, styles.w100, styles.mt5]}
-                    >
-                        <Text style={[styles.buttonText, styles.buttonSuccessText]}>
-                            Create Group
-                        </Text>
-                    </TouchableOpacity>
+                    <View style={[styles.p5, styles.pt2]}>
```

Comment:
Can we try this with no padding on the top?

Reply:
Done!